src/main/java/microsoft/exchange/webservices/data/FileAttachment.java

```diff
@@ -218,8 +218,13 @@ public void load(String fileName) throws Exception {
     this.loadToStream = fos;
```

Comment:
looking at the code I saw 
```java
    FileOutputStream fos = new FileOutputStream(fileStream);
    this.loadToStream = fos;
```
fos is only used to be assigned but the following should also work?
```java
this.loadToStream = new FileOutputStream(fileStream);
```